src/popup.ts

```diff
@@ -1,4 +1,11 @@
 const showClickablesCheckbox = document.getElementById("showClickables") as HTMLInputElement;
+const reconnectButton = document.getElementById("reconnect") as HTMLAnchorElement;
+
+async function reconnect() {
```

Comment:
I think I missed this file in the first review, but lets be consistent about using this syntax for lambdas:

```
(settings) => {
}
```

for each function that we only call once, it's also cleaner to just inline them, so we can more clearly see the events they're attached to (i.e., we want to minimize the distance between a variable and where it's used).